apps/mull-api/uploads/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.jpg
 *.jpeg
 *.png
+*.gif
```

Comment:
Could you also add `*.webp` while you're at it 👀

Reply:
Done!